test/test_testing.py

```diff
@@ -6668,8 +6669,8 @@ def _on_results_action(self, event: ops.ActionEvent):
             unobserved-param-tester:
               description: consectetur adipiscing elit
               params:
-                foo
-                bar
+                foo: {}
+                bar: {}
```

Comment:
https://matrix.to/#/!eNCNzcteYUDDYpStsu:ubuntu.com/$noL-ohAQxPopkDNj9U9id7P6_JPYqb2z-V1I_eFn42I?via=ubuntu.com&via=matrix.org&via=laquadrature.net

Reply:
YAML wasn't conformant to metadata schema before.
Bad value type wasn't the intention of the test.
There's a chance that YAML just wasn't formatted correctly in the first place.